packages/legacy-scripting-runner/bundle.js

```diff
@@ -224,6 +224,7 @@ const convertBundleMeta = (meta) => {
       meta.first_poll === undefined ? meta.isPopulatingDedupe : meta.first_poll,
     limit: meta.limit,
     page: meta.page,
+    isBulkRead: meta.isBulkRead,
```

Comment:
you might do `meta.isBulkRead || false` so that it's always present, but reacts accordingly if `isBulkRead` isn't in the incoming meta. IIRC we're only supplying that to certain apps to start.